internal/staging/stdaddr/README.md

```diff
@@ -182,6 +183,24 @@ if an address can be converted to its public key hash variant by type asserting
 the interface and then convert it by making use of the `AddressPubKeyHash`
 method provided by the interface.
 
+### Obtaining Serialized Public Key From Public Key Addresses
+
+When making use of public key address, callers often need to obtain the
```

Comment:
s / ... making use of public key address, / ... making use a public key address,

Reply:
addresses?

Reply:
Good catch.  I had addresses in mind there and wrote it singular.  Updated.